packages/calcite-components/conventions/Styling.md

```diff
@@ -41,50 +43,63 @@ In the [global CSS file](https://github.com/Esri/calcite-design-system/blob/main
 
 ```scss
-// 🙅‍♀️ using the sass var will not correctly inherit or change in light/dark mode
+// 👍 Using the CSS var will inherit the value correctly
 :host {
-  color: $ui-brand-light;
+  color: var(--calcite-color-brand);
 }
+```
+
+```scss
+@import "~@esri/calcite-design-tokens/dist/scss/light";
 
-// 👍 using the CSS var will inherit correctly
+// Using the sass var will set the color of host to the value of the brand color in light mode
+// 🙅‍♀️ However, it will not correctly inherit or change it's value when swapping light/dark mode
 :host {
-  color: var(--calcite-color-brand);
+  color: $calcite-color-brand;
 }
 ```
 
+## Legacy Tokens
+
+In the release of 2.0 Calcite Component styles got a major refactor which included the removal and reassignment of legacy CSS Custom Properties originally introduced through calcite-styles/calcite-colors. To see a full list of CSS Custom Property additions, deletions, and renamed tokens please refer to the [Calcite Design Tokens 2.0 Changelog > Map of token changes](../../calcite-design-tokens/CHANGELOG.md#20-map-of-token-changes).
+
+For backwards compatibility, deprecated tokens will continue to be provided until the next major release via [\_legacy.scss](../src/assets/styles/_legacy.scss)
+
 ## Custom Themes
 
-Since Calcite Components might be used in many different contexts such as configurable apps, multiple themes and appearances need to be supported. The most common use case for custom themes are applications where the end user needs to be able to customize brand colors and typography. To this end custom theming can be accomplished by overriding the [CSS Custom Properties (CSS Variables)](https://developer.mozilla.org/en-US/docs/Web/CSS/--*) from the main light and dark modes with new values:
+Since Calcite Components might be used in many different contexts such as configurable apps, multiple themes and appearances need to be supported. The most common use case for custom themes are applications where the end-user needs to be able to customize brand colors and typography. To this end, custom theming can be accomplished by overriding the [CSS Custom Properties (CSS Variables)](https://developer.mozilla.org/en-US/docs/Web/CSS/--*) from the main light and dark modes with new values:
 
 ```css
-:root {
-  --calcite-color-brand: red;
+@media (prefers-color-scheme: light) {
+  :root {
+    --calcite-color-brand: red;
+  }
 }
 ```
 
-You can apply these overrides to individual components as well:
+You can override tokens on specific Calcite Components:
 
 ```css
 calcite-slider {
-  --calcite-color-brand: red;
+  --calcite-color-brand: blue;
 }
 ```
 
-Or, add a class to the specific instance:
+Or, override it in a class:
 
 ```css
 .my-custom-theme {
-  --calcite-color-brand: red;
+  --calcite-color-brand: green;
 }
 ```
 
+Additionally, inline styling on a particular instance can be achieved:
```

Comment:
nitpick: Remove extraneous "on a particular instance" language per [this review comment](https://github.com/Esri/calcite-design-system/pull/8357#discussion_r1418071444).